app/assets/javascripts/admin/addon/templates/components/admin-penalty-similar-users.hbs

```diff
@@ -0,0 +1,35 @@
+<div class="penalty-similar-users">
+  <p class="alert alert-danger">
+    {{i18n "admin.user.other_matches" (hash count=this.user.similar_users_count username=this.user.username)}}
+  </p>
+
+  <table class="table">
+    <thead>
+      <tr>
+        <th></th>
+        <th>{{i18n "username"}}</th>
+        <th>{{i18n "last_seen"}}</th>
+        <th>{{i18n "admin.user.topics_entered"}}</th>
+        <th>{{i18n "admin.user.posts_read_count"}}</th>
+        <th>{{i18n "admin.user.time_read"}}</th>
+        <th>{{i18n "created"}}</th>
+      </tr>
+    </thead>
+
+    <tbody>
+      {{#each this.user.similar_users as |user|}}
+        <tr>
+          <td>
+            <Input @type="checkbox" disabled={{not (get user this.penaltyField)}} {{on "click" (action "selectUserId" user.id)}} />
```

Comment:
Any chances you could add a tooltip to explain why the checkbox is disabled?

Reply:
At this moment, the guardian method is quite simple so the only reason a person cannot be silenced or suspended is that they are staff. If this changes in the future then this can go out of sync so I believe the reason should come from the server side. I can follow up on this in a different PR.